add gps and timestamp examples

etrombly opened the pull request:
Added two quick examples. Should be enough to get people started.

felixc commented on examples/timestamp.rs:
```diff
@@ -0,0 +1,20 @@
+fn main() {
+    if let Ok(meta) = Metadata::new_from_path("examples/DSCN0010.jpg") {
+        if let Ok(time) = meta.get_tag_string("Exif.Image.DateTime") {
+            if let Ok(timestamp) = NaiveDateTime::parse_from_str(&time, "%Y:%m:%d %H:%M:%S") {
```
It's definitely more practical for actual use, but should we bother bringing in an extra dev dependency for the example?

felixc commented on examples/timestamp.rs:
```diff
@@ -0,0 +1,20 @@
+            meta.save_to_file("examples/DSCN0010.jpg").expect("Couldn't save metadata to file");
+        }
+    }
+}
```
Suuuper trivial optional nitpick: Could you add a final newline to these? It just makes it nicer for line-editing tools like `sed`, or if you `cat` the file to see what's in it.

etrombly commented:
Sorry it took so long to respond, didn't notice you commented. I removed the chrono dependency, and added the newlines. Also changed out the example image to one that explicitly says it can be used.

felixc commented:
Merged in https://github.com/felixc/rexiv2/commit/0e49bafd8968179fe5ac5138d3acc311724fd022 !! (Don't pay attention to GitHub saying it's not merged; it's just 'cause I edited the commit to add you to the list of contributors :)

Thank you again for contributing, and sorry it fell off my radar for so long.
